tests/projects/test_project.py

```diff
@@ -1477,3 +1477,30 @@ def test_load_project_from_yaml_with_function(context):
             )
             == {}
         )
+
+
+def test_project_create_remote():
+    # test calling create_remote without git_init=True on project creation
+
+    # Set the current working directory to a temporary directory
+    temp_dir = tempfile.mkdtemp()
```

Comment:
can do it with context manager for easier cleanup:
~~~
with tempfile.TemporaryDirectory() as tmp_dir:
~~~